pom.xml

```diff
@@ -378,6 +378,7 @@
                         <systemPropertyVariables>
                            <!-- See test org.hibernate.ogm.test.utils.EnvironmentTest -->
                            <hibernate.service.allow_crawling>false</hibernate.service.allow_crawling>
+                           <user.timezone>UTC</user.timezone>
```

Comment:
Ofcource!  If I not set UTC timezone  then test is broken. Reason: Moscow timezone is  UTC+3